docs/rules/sort-amd-paths.md

```diff
@@ -16,24 +16,31 @@ Determines which part (or parts) of the path will be used to sort by. The possib
 * `fullpath` - Sorts by the full path, including the module file name.
 * `basename` - Sorts by module file name only, ignoring the directory name.
 
+#### `sortPlugins`
+
+Determines how paths with plugin prefix will be sorted. The possible values are:
+
+* `preserve` (default) - Behaves like there is no plugin prefix before the path. Prefix is a part of a path.
```

Comment:
This could benefit from some clarification:
```md
* `preserve` (default) - Treats plugin prefix as part of full path when sorting."
* `ignore` - Ignores plugin prefix completely when sorting."
...
```